FiveCalls/FiveCalls/IssueDetailViewController.swift

```diff
@@ -27,6 +28,8 @@ class IssueDetailViewController : UIViewController, IssueShareable {
         tableView.estimatedRowHeight = 100
         tableView.rowHeight = UITableViewAutomaticDimension
         NotificationCenter.default.addObserver(self, selector: #selector(madeCall), name: .callMade, object: nil)
+
+        Mixpanel.sharedInstance()?.track("Topic", properties: ["IssueID": self.issue.id])
```

Comment:
What would you say to an extension on `UIViewController` like this:

```swift
extension UIViewController {
    func trackEvent(_ name: String, properties: [String:String]) {
        Mixpanel.sharedInstance()?.track(...)
    }
}
```

Reply:
This way we decouple the actual type (and service) used to track these events and cleans up the callsite a bit.

Reply:
👍